studio/src/pages/[organizationSlug]/[namespace]/graph/[slug]/changelog/index.tsx

```diff
@@ -320,7 +320,11 @@ const ChangelogPage: NextPageWithLayout = () => {
           }
           actions={
             <CLI
-              command={`npx wgc subgraph publish users --namespace ${router.query.namespace} --schema users.graphql`}
+              command={
+                graphData?.graph?.type === "federated"
+                  ? `npx wgc subgraph publish <subgraph-name> --namespace ${router.query.namespace} --schema <path-to-schema>`
```

Comment:
Same here. Provide a dropdown.

Reply:
This one shouldn't be a dropdown. We only show one of them.

Reply:
ok